src/UI/Badge.elm

```diff
@@ -69,27 +73,57 @@ type alias Properties =
     }
 
 
+type alias Options =
+    { withIcon : Maybe Icon
+    }
+
+
 type BadgeTone
     = ToneGray
     | TonePrimary
     | ToneDanger
     | ToneWarning
     | ToneSuccess
+    | ToneClear
 
 
 type BadgeBrightness
     = Light
     | Dark
 
 
+{-| A variation of the badge with outline for light backgrounds.
+
+    Badge.outlineLight "SENT"
+
+-}
+outlineLight : String -> Badge
+outlineLight content =
+    Options Nothing
+        |> Badge { content = content, tone = ToneClear, brightness = Light }
+
+
+outlineLightWithIcon : String -> Icon -> Badge
+outlineLightWithIcon content icon =
+    Options (Just icon)
+        |> Badge { content = content, tone = ToneClear, brightness = Light }
+
```

Comment:
nope, this is wrong, the idea is to have a single exposed function with this signature:
```elm
withIcon : Icon -> Badge -> Badge
```

Reply:
Thanks @PedroHLC , this is an awesome idea...totally life changing advice for a beginner 😄  !